invenio_jobs/models.py

```diff
@@ -63,6 +70,13 @@ def parsed_schedule(self):
         elif stype == "interval":
             return timedelta(**schedule)
 
+    def render_args(self, **ctx):
+        """Render job arguments."""
+        args = deepcopy(self.default_args)
+        ctx.setdefault("job", self)
+        walk_dict_values(args, lambda val: eval_tpl_str(val, ctx))
```

Comment:
The idea is to render the arguments into a jinja template I assume? 

is it ok for  this logic be so close to the model?

Reply:
In the end, I added a `Run.create` classmethod, that also takes care of some of the implicit logic we were doing in the service and scheduler (e.g. setting the default queue).

I think we can still get away without an API layer, so keeping things in the model is somewhat fine. I've also made the conscious decision that template arguments are evaluated before a Run is created, and the results are stored as is. That means that a Run is "pre-baked" with whatever args were computed at the time it's created.